specs/binding-tls.spec/src/main/scripts/io/aklivity/zilla/specs/binding/tls/schema/tls.schema.patch.json

```diff
@@ -229,14 +239,15 @@
                         "type": "array",
                         "items":
                         {
-                            "type": [ "string", "null" ]
+                            "type": "string"
                         }
                     },
                     "mutual":
                     {
                         "title": "Mutual Authentication",
                         "type": "string",
-                        "enum": [ "required", "requested", "none" ]
+                        "enum": [ "required", "requested", "none" ],
+                        "default": "none"
```

Comment:
The default is dependent on the existence of `trust` peer field. If `trust` is _not_ specified, then default is `none`, but if `trust` _is_ specified, then default is `required`.

https://github.com/aklivity/zilla/blob/f9222ff6b710c9873fa8ee7445934c99bf0b74fe/runtime/binding-tls/src/main/java/io/aklivity/zilla/runtime/binding/tls/config/TlsOptionsConfigBuilder.java#L109-L114